src/functions/template-tags/html.php

```diff
@@ -127,8 +127,79 @@ function tribe_disabled( $disabled, $echo = true ) {
 
 		if ( ! $echo ) {
 			return $disabled_string;
-		} else {
-			echo $disabled_string;
 		}
+
+		echo $disabled_string;
+	}
+}
+
+/**
+ * Adds rel attribute for links.
+ *
+ * @since TBD
+ *
+ * @param string|array $rel  Value(s) for rel attribute.
+ * @param boolean      $echo Whether to echo the string or return it.
+ *
+ * @return string The rel attribute.
+ */
+function tribe_rel( $rel, $echo = true ) {
+	if ( empty( $rel ) ) {
+		return;
+	}
+
+	if ( is_array( $rel )) {
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: There must be a single space before the last parenthesis
* :no_entry_sign: There must be a single space before the last parenthesis
* :no_entry_sign: No space before closing parenthesis is prohibited
* :no_entry_sign: No space before closing parenthesis is prohibited

<sub>hash: 3f6245db7df64be2ed32a1e68a778e2e</sub>